common/src/main/java/com/wynntils/models/beacons/type/BeaconColor.java

```diff
@@ -5,34 +5,43 @@
 package com.wynntils.models.beacons.type;
 
 import com.wynntils.models.activities.type.ActivityType;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.colors.CustomColor;
 import net.minecraft.world.item.Item;
 import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.item.Items;
 
 public enum BeaconColor {
-    GREEN(25, Items.GOLDEN_SHOVEL, true, ActivityType.STORYLINE_QUEST),
-    PINK(24, Items.GOLDEN_SHOVEL, false, ActivityType.BOSS_ALTAR),
-    YELLOW(3, Items.GOLDEN_PICKAXE, true, ActivityType.RAID),
-    BLUE(4, Items.GOLDEN_PICKAXE, true, ActivityType.LOOTRUN_CAMP),
-    PURPLE(5, Items.GOLDEN_PICKAXE, true, ActivityType.MINI_QUEST),
-    GRAY(6, Items.GOLDEN_PICKAXE, true, null),
-    ORANGE(7, Items.GOLDEN_PICKAXE, true, ActivityType.CAVE),
-    RED(8, Items.GOLDEN_PICKAXE, true, ActivityType.DUNGEON),
-    DARK_GRAY(9, Items.GOLDEN_PICKAXE, true, null),
-    WHITE(10, Items.GOLDEN_PICKAXE, true, ActivityType.WORLD_DISCOVERY), // or ActivityType.TERRITORIAL_DISCOVERY
-    AQUA(11, Items.GOLDEN_PICKAXE, true, ActivityType.QUEST), // This is CYAN in the resource pack
-    RAINBOW(12, Items.GOLDEN_PICKAXE, true, null);
+    GREEN(25, Items.GOLDEN_SHOVEL, true, ActivityType.STORYLINE_QUEST, CommonColors.GREEN),
+    PINK(24, Items.GOLDEN_SHOVEL, false, ActivityType.BOSS_ALTAR, CommonColors.PINK),
+    YELLOW(3, Items.GOLDEN_PICKAXE, true, ActivityType.RAID, CommonColors.YELLOW),
+    BLUE(4, Items.GOLDEN_PICKAXE, true, ActivityType.LOOTRUN_CAMP, CommonColors.BLUE),
+    PURPLE(5, Items.GOLDEN_PICKAXE, true, ActivityType.MINI_QUEST, CommonColors.PURPLE),
+    GRAY(6, Items.GOLDEN_PICKAXE, true, null, CommonColors.GRAY),
+    ORANGE(7, Items.GOLDEN_PICKAXE, true, ActivityType.CAVE, CommonColors.ORANGE),
+    RED(8, Items.GOLDEN_PICKAXE, true, ActivityType.DUNGEON, CommonColors.RED),
+    DARK_GRAY(9, Items.GOLDEN_PICKAXE, true, null, CommonColors.DARK_GRAY),
+    WHITE(
```

Comment:
What about those long lines..? ;-) (maybe move the // to a separate line)

Reply:
The downside of auto formatters..

Reply:
Actually.. This comment can be removed. Because the code needs to adapt. It is an edge case but we use this to get the color for activity beacons..

Reply:
It is sad bit it might have to stay like this.